public/app/features/admin/UserListAdminPage.tsx

```diff
@@ -82,7 +83,7 @@ const UserListAdminPageUnConnected = ({
   return (
     <Page.Contents>
       <div className="page-action-bar" data-testid={selectors.container}>
-        <div className="gf-form gf-form--grow">
+        <InlineField grow="true">
```

Comment:
```suggestion
        <InlineField grow>
```
`grow` is a boolean prop. If you run a linter, it will tell you.